src/stores/PlaceStore.ts

```diff
@@ -28,13 +28,61 @@ export const usePlaceStore = defineStore('placeStore', () => {
     details.placeType = 'onsen';
     details.location = { latitude: 35.65867467719728, longitude: 139.72758253068733 };
     details.etiquettes = [
-      { id: 1, label: 'No smoking', status: 'allowed' },
-      { id: 2, label: 'No tattoos', status: 'allowed' },
-      { id: 3, label: 'Towels provided', status: 'allowed' },
-      { id: 4, label: 'No swimming', status: 'allowed' },
+      { id: 1, label: 'Smoking', status: 'not-allowed' },
+      { id: 2, label: 'Tattoos', status: 'not-allowed' },
+      { id: 3, label: 'Towels', status: 'allowed' },
+      { id: 4, label: 'Swimming', status: 'not-allowed' },
```

Comment:
Brilliant, I made the same update. At some point, we have to fetch these from the database, but I'm not sure if we have finalized the etiquettes.